Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -126,6 +126,12 @@ public Element build(final CallbackProperties properties) throws SAMLAssertionBu
             final Subject subject = createSubject(properties, certificate, publicKey);
             assertion.setSubject(subject);
 
+            // add conditions statement
+            if (isConditionsDefaultValueEnabled()) {
```

Comment:
This isn't right.  Default enable only happens if the values passed in aren't valid.  Conditions should be created (value or not), if some sort of value is passed in via the assertion block.